style-dictionary/visual-refresh/metadata/colors.ts

```diff
@@ -470,6 +470,16 @@ const metadata: StyleDictionary.MetadataIndex = {
     themeable: true,
     public: true,
   },
+  colorTextLayoutToggle: {
+    description: "This is the default color of the app layout toggle button when it's active.",
+    public: true,
+    themeable: true,
+  },
+  colorTextLayoutToggleSelected: {
```

Comment:
Why are these called `colorText*`? That seems misleading since theyre not text colors, right?

Reply:
Good question. The name is not new as of this PR so I can't speak to the reasoning. I would guess because it uses the `color` property for the text and the SVG inherits it using `currentColor`.

Reply:
I checked and it follows the same convention as our other button tokens...`colorText[Button][Variant][State]`.

Reply:
gotcha, okay.